sdk/core/azure-core-tracing-opentelemetry/samples/sample_eventgrid.py

```diff
@@ -0,0 +1,57 @@
+"""
+Examples to show usage of the azure-core-tracing-opentelemetry
+with the storage SDK.
+
+This example traces calls for publishing cloud data and exports it
+using the ConsoleSpanExporter.
+
+An alternative path to export using AzureMonitor is also mentioned in the sample. Please take
+a look at the commented code.
+"""
+
+# Declare OpenTelemetry as enabled tracing plugin for Azure SDKs
+from azure.core.settings import settings
+from azure.core.tracing.ext.opentelemetry_span import OpenTelemetrySpan
+
+settings.tracing_implementation = OpenTelemetrySpan
+
+# In the below example, we use a simple console exporter, uncomment these lines to use
+# the Azure Monitor Exporter.
+# Example of Azure Monitor exporter, but you can use anything OpenTelemetry supports
+# from azure_monitor import AzureMonitorSpanExporter
+# exporter = AzureMonitorSpanExporter(
```

Comment:
Please add a ref to the package name on PyPI as well, so people know what they need to install.